Formula/m/mariadb.rb

```diff
@@ -1,10 +1,9 @@
 class Mariadb < Formula
   desc "Drop-in replacement for MySQL"
   homepage "https://mariadb.org/"
-  url "https://downloads.mariadb.com/MariaDB/mariadb-11.0.2/source/mariadb-11.0.2.tar.gz"
-  sha256 "3c71576cad0ea416882270e383f94cc89693b7fbcce1fa4f306fe3e931e464ae"
-  license "GPL-2.0-only"
-  revision 1
+  url "https://archive.mariadb.org/mariadb-11.1.2/source/mariadb-11.1.2.tar.gz"
+  sha256 "19a9e980e57fa332931f643b48ad7390528c889ff6ea8b0e16fd306aa3088238"
+  license "GPL-2.0-or-later"
```

Comment:
Corrected the license; see https://github.com/MariaDB/server/blob/mariadb-11.1.2/COPYING#L299

Reply:
@ZhongRuoyu The "or...later" text in the GPL `COPYING` file is provided as an example and isn't an indication of whether the project is `-only` or `-or-later`. Instead, it's necessary to look through other files (e.g., `README`, leading comments in source files, etc.) to see if they use the "or...later" language. Some projects clearly state that the license is only a specific GPL version but others are implicitly "only" through absence of the "or...later" language in license text [outside of the `COPYING` file].

In this case, the `README` contains a "Licensing" section that explicitly states:

> MariaDB is specifically available only under version 2 of the GNU General Public License (GPLv2). (I.e. Without the "any later version" clause.) This is inherited from MySQL. Please see the README file in the MySQL distribution for more information.

With that in mind, the license for the mariadb formulae should be reverted back to `GPL-2.0-only`.

Reply:
Sorry for the oversight @samford. Have opened #140689 to correct the other `mariadb` formulae; will update the remaining soon.